Samples/Android/Speech Devices SDK Starter App/example/app/src/main/java/com/microsoft/coginitiveservices/speech/samples/sdsdkstarterapp/conversation.java

```diff
@@ -0,0 +1,388 @@
+import android.text.Layout;
+import android.text.TextUtils;
+import android.text.method.ScrollingMovementMethod;
+import android.util.Log;
+import android.util.Pair;
+import android.view.Menu;
+import android.view.MenuItem;
+import android.view.WindowManager;
+import android.widget.ListView;
+import android.widget.TextView;
+
+import com.microsoft.cognitiveservices.speech.ResultReason;
+import com.microsoft.cognitiveservices.speech.SpeechConfig;
+import com.microsoft.cognitiveservices.speech.audio.AudioConfig;
+import com.microsoft.cognitiveservices.speech.conversation.ConversationTranscriber;
+import com.microsoft.cognitiveservices.speech.conversation.ConversationTranscriptionEventArgs;
+import com.microsoft.cognitiveservices.speech.conversation.Participant;
+import com.microsoft.cognitiveservices.speech.conversation.User;
+
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.math.BigInteger;
+import java.net.URI;
+import java.util.ArrayList;
+import java.util.Calendar;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Properties;
+import java.util.Random;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.concurrent.Future;
+
+public class conversation extends AppCompatActivity {
+    private HashMap<String, String> signatureMap = new HashMap<>();
+    private HashMap<String, Integer> colorMap = new HashMap<>();
+    private TextView IntermediateTextView;
+    private static final String CTSKey = "<Conversation Transcription Service Key>";
+    private SpeechConfig speechConfig = null;
+    private static final String inroomEndpoint = "<Conversation Transcription Service Endpoint>";
+    private final ArrayList<String> attendeeName = new ArrayList<>();
+    private final String logTag = "CTS";
+    private boolean meetingStarted = false;
+    private ConversationTranscriber transcriber = null;
+    private final HashMap<Pair<String, BigInteger>, ConversationTranscriptionEventArgs> transcriptions = new HashMap<>();
+    private TranscriptionAdapter transcriptionAdapter;
+    private ListView transcriptionView;
+
+    public boolean onCreateOptionsMenu(Menu menu)
+    {
+        getMenuInflater().inflate(R.menu.ctsmenu, menu);
+        return true;
+    }
+
+    public boolean onOptionsItemSelected(MenuItem item)
+    {
+        switch (item.getItemId())
+        {
+            case R.id.back:
+            {
+                startActivity(new Intent(getApplicationContext(), MainActivity.class).setFlags(Intent.FLAG_ACTIVITY_CLEAR_TOP));
+                return true;
+            }
+            case R.id.startOrStopMeeting:
+            {
+                if (meetingStarted)
+                {
+                    stopClicked(item);
+                    return true;
+                }
+                clearTextBox();
+                speechConfig = SpeechConfig.fromEndpoint(URI.create(inroomEndpoint), CTSKey);
```

Comment:
Please use fromSubscription as that is now supported in 1.5.1 version of Speech SDK.
With that you need to use eastasia or centralus key as documented in how to guide

Reply:
I will update the modification on next release which is based on carbon V.1.51

Reply:
fixed